File: src/Persona.cpp
#include "../headers/persona.h"
#include <iostream>

using namespace std;
using namespace Modelo;

void Persona::MostrarInfo()
{
	cout<<"Nombre: "<<Persona::get_nombres()<<endl;
	cout<<"Edad: "<<Persona::get_edad()<<endl;
}
string Persona::get_nombres()
{
	return nombres;
}
int Persona::get_edad()
{
	return edad;
}
void Persona::set_edad(int edad_persona)
{
	edad=edad_persona;
}
void Persona::set_nombres(string nombres_persona)
{
	nombres=nombres_persona;
}

File: java/src/main/java/com/upn/models/Persona.java
package com.upn.models;

public class Persona {
    private int id;
    private String nombre;
    private int edad;

    public void setId(int id) {
        this.id = id;
    }

    public Persona() {
    }

    public Persona(int id, int edad, String nombre) {
        this.id = id;
        this.edad = edad;
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public int getId() {
        return id;
    }

    public int getEdad() {
        return edad;
    }

    @Override
    public String toString() {
        return "Persona{" +
                "id=" + id +
                ", nombre='" + nombre + '\'' +
                ", edad=" + edad +
                '}';
    }
}
